### ISSUE TITLE
hasMany array is lost

### ISSUE THREAD

**Issue by tute** [kurko/ember-sync#15]
If I define my `DS.hasMany('child')` as `DS.attr('array')` (I've got a trivial array `Transform`), then in ember-sync I can see the array of child ids in the parent object.

When I leave the `hasMany` call, then the attribute gets `undefined`. Debugging `lib/ember-sync/query.js`:
```js
onlinePromise.then(function(results) {
          // ...
          results.forEach(function(onlineResult) {
                    onlineResult.get('childs') // => undefined
```
My `onlineStore` is the same as my ex-RESTAdapter, only moved to plugin `ember-sync` in the middle. API has no changes. Why could this be happening? Where should I continue digging into this issue?

**Comment by kurko** [comment]
I'm experiencing this in my app now. I'll try to come up with a path.

**Comment by kurko** [comment]
@tute any chance you can test this again? I believe I have fixed this in ca9797661e253842389d04056242aff3ea22eda9. If it doesn't work for you, please reopen the issue ;)
